src/codeu/chat/server/Server.java

```diff
@@ -30,6 +29,7 @@
 import codeu.chat.common.Secret;
 import codeu.chat.common.User;
 import codeu.chat.common.ServerInfo;
+//FIX THIS. I don't know why IntelliJ does this automatically when you have a ton of imports.
```

Comment:
I don't think it's that. It's something that I need to change in the ItelliJ settings. Are you guys having the same problem? 😣